libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -186,6 +186,27 @@ def _llm_type(self) -> str:
         """Return type of chat model."""
         return "anthropic-chat"
 
+    @property
+    def lc_secrets(self) -> Dict[str, str]:
+        return {"anthropic_api_key": "ANTHROPIC_API_KEY"}
+
+    @classmethod
+    def is_lc_serializable(cls) -> bool:
+        return True
```

Comment:
don't we still need to manually specify the path?

Reply:
good call